cypress/e2e/19-execution.cy.ts

```diff
@@ -536,10 +536,14 @@ describe('Execution', () => {
 
 		cy.wait('@workflowRun').then((interception) => {
 			expect(interception.request.body).not.to.have.property('runData').that.is.an('object');
-			expect(interception.request.body).to.have.property('pinData').that.is.an('object');
+			expect(interception.request.body).not.to.have.property('pinData').that.is.an('object');
```

Comment:
do we really need this assertion? I'd say this is only needed if there is a risk that someone might add the `pinData` field back. Which I don't think we need to put a check for.